packages/cli/src/ui/utils/clipboardUtils.ts

```diff
@@ -86,30 +92,62 @@ export async function saveClipboardImage(
 
       const { stdout } = await execAsync(`osascript -e '${script}'`);
 
+        if (stdout.trim() === 'success') {
+          // Verify the file was created and has content
+          try {
+            const stats = await fs.stat(tempFilePath);
+            if (stats.size > 0) {
+              return tempFilePath;
+            }
+          } catch {
+            // File doesn't exist, continue to next format
+          }
+        }
+
+        // Clean up failed attempt
+        try {
+          await fs.unlink(tempFilePath);
+        } catch {
+          // Ignore cleanup errors
+        }
+      }
+    } else if (process.platform === 'win32') {
+      // Windows: Use PowerShell to save clipboard image
+      const tempFilePath = path.join(tempDir, `clipboard-${timestamp}.png`);
+      
+      // Use a different approach to avoid quote escaping issues
+      const escapedPath = tempFilePath.replace(/\\/g, '\\\\').replace(/"/g, '`"');
+      
+      const { stdout } = await execAsync(
+        `powershell -command "Add-Type -AssemblyName System.Windows.Forms; Add-Type -AssemblyName System.Drawing; if ([System.Windows.Forms.Clipboard]::ContainsImage()) { $image = [System.Windows.Forms.Clipboard]::GetImage(); $image.Save('${escapedPath}', [System.Drawing.Imaging.ImageFormat]::Png); Write-Output 'success' } else { Write-Output 'error' }"`,
+      );
+
       if (stdout.trim() === 'success') {
```

Comment:
the logic from line 125 on seems duplicated with the mac case. please merge the two.